.github/workflows/integration.yml

```diff
@@ -1,23 +0,0 @@
-name: Integration
-on:
-  pull_request:
-    branches:
-    - master
-    - release-*
-    - ipv6
```

Comment:
you need to keep this file and remove `ipv6` from the list of branches

Reply:
Updated.